src/server/Document/Renderers/litePageTransforms/transformTagsWithinHead.ts

```diff
@@ -0,0 +1,25 @@
+export default (html: string) => {
+  let modifiedHtml = html;
+
+  try {
+    const tags = modifiedHtml.match(/<[^>]*>/g) || [];
+
+    tags.forEach(tag => {
+      const dataReactHelmetAttribute = tag?.includes(
+        'data-react-helmet="true"',
+      );
+
+      if (dataReactHelmetAttribute) {
```

Comment:
Do we even need this if statement? Or is it more performant to only use string.replace when required?